### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/echarts.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#567] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/echarts.html

让你 VuePress 站点中的 Markdown 文件支持图表。 此插件使用 ECharts (https://echarts.apache.org/zh/index.html) 提供相应功能。 配置 tab TS tab JS 格式 使用 JSON 如果你可以很轻松的生成数据，你可以直接通过一个 JSON 代码块来提供 Echarts 配置: 使用...

https://plugin-md-enhance.vuejs.press/zh/guide/echarts.html

**Comment by lingdu990130556** [comment]
怎么支持中国地图啊

**Reply by Mister-Hope** [reply]
按照echarts官网的提示自己做适配，不能用link标签在脚本中用 fetch 获取

**Reply by lingdu990130556** [reply]
官网都看不懂。。。
问chatGPT3.5的也是不成功，
希望你的这个文档补充一下引入地图的相关的案例
[地址](http://lingdu_dou.gitee.io/vuepress-lingdu-v2/statstics/)

**Reply by Mister-Hope** [reply]
我认为这应该是你的问题，一个基于 Echarts 的插件文档没有义务给你来一个 echarts 教程....你应该自行通过寻找类似的 Echarts 教程解决。

你要搞清楚只有你会用 Echarts 实现一个功能，但使用插件你不会了，这才是我的问题。

**Reply by lingdu990130556** [reply]
😂好的，知道了，我再看看
